### PR TITLE
Portal updates for ssha 0.2

### PR DESCRIPTION
**Pull request by NouhaBoujelben**

### CODE REVIEW

File: Portal/vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/client/bean/SSH.java [hunk 1]
```diff
@@ -35,28 +35,49 @@
...
     private String name;
     private String user;
     private String host;
-    private String port;
+    private int port;
+    private TransfertType transfertType;
     private String directory;
     private String status;
+    private long numberSynchronizationFailed;
+    private boolean deleteFilesFromSource;
```
Comment (glatard) [inline Portal/vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/client/bean/SSH.java]: Add theEarliestNextSynchronistation.

File: Portal/vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/client/view/ssh/EditSSHLayout.java [hunk 2]
```diff
@@ -87,7 +93,21 @@ private void configure() {
         portField = FieldUtil.getTextItem(450, null);
         directoryField = FieldUtil.getTextItem(450, null);
         statusField = FieldUtil.getTextItem(450, null);
-        
+        numberSynchronizationFailedField = FieldUtil.getTextItem(450, null);
```
Comment (glatard) [inline Portal/vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/client/view/ssh/EditSSHLayout.java]: Disable this field and earliestNextSynchronization date. Add a redmine ticket to reset these 2 fields.

File: Portal/vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/server/dao/MySQLDAOFactory.java [hunk 3]
```diff
@@ -57,8 +57,8 @@ private MySQLDAOFactory()  {
          try {
              logger.info("Configuring VIP SSH database.");
              PlatformConnection.getInstance().createTable("VIPSSHAccounts","email VARCHAR(255), LFCDir VARCHAR(255), "
-                     + "sshUser VARCHAR(255), sshHost VARCHAR(255), sshDir VARCHAR(255), sshPort INT, validated BOOLEAN,"
-                     + " auth_failed BOOLEAN, PRIMARY KEY(email,LFCDir), FOREIGN KEY (email) REFERENCES VIPUsers(email) "
+                     + "sshUser VARCHAR(255), sshHost VARCHAR(255), sshDir VARCHAR(255), sshPort INT, validated BOOLEAN, "
+                     + "auth_failed BOOLEAN, theEarliestNextSynchronistation TIMESTAMP, numberSynchronizationFailed BIGINT, transfertType VARCHAR(255), deleteFilesFromSource BOOLEAN DEFAULT 0, PRIMARY KEY(email,LFCDir), FOREIGN KEY (email) REFERENCES VIPUsers(email) "
```
Comment (glatard) [inline Portal/vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/server/dao/MySQLDAOFactory.java]: Split the request line.

File: Portal/vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/client/bean/SSH.java (hunk 1 above)
Comment (NouhaBoujelben) [inline Portal/vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/client/bean/SSH.java]: done

File: Portal/vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/server/dao/MySQLDAOFactory.java (hunk 3 above)
Comment (NouhaBoujelben) [inline Portal/vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/server/dao/MySQLDAOFactory.java]: done

File: Portal/vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/client/view/ssh/EditSSHLayout.java (hunk 2 above)
Comment (NouhaBoujelben) [inline Portal/vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/client/view/ssh/EditSSHLayout.java]: i removed this field from editSSH Layout

**Pull request merged**